lib/CoverageGenerator.js

```diff
@@ -0,0 +1,71 @@
+/**
```

Comment:
euh.. this file can probably be removed. It was a relic from earlier attempts. I'll check it out an delete it unless it's still being used...